src/sas/qtgui/Utilities/DocViewWidget.py

```diff
@@ -21,6 +24,39 @@
 '''
 
 
+class DocGenThread(CalcThread):
+    """Thread performing the fit """
```

Comment:
Are those changes related to the issue of broken Linux testing? Or just a stealthy fix for doc gen? :)